### PR TITLE
Unit test documentation

### LINKED ISSUE #203: Need mark document explaining our convention for how unit tests are implemented in eMach
We need a Markdown document that either explains the convention that we are using for creating unit tests and how unit tests should be run. This can simply be a link to guidelines someone else has written (if such a web site exists for our needs) or it can be a detailed explanation of our implementation

### PR DESCRIPTION
**Pull request by aimthiazz**

### CODE REVIEW

**Review by elsevers — CHANGES_REQUESTED** [review]
@aimthiazz Thanks. Looking good. A few things:

You seem to have two identical markdown documents in here... I guess, delete one?

While your markdown document has the correct general content, it is not quite written from the perspective I have requested. I want this document to accomplish two things:

1. It should set our unit testing policy. You need to start by telling us that eMach unit testing for Python will use the unittest framework. Then tell us guidelines/policies, etc. 

2. It should provide some basic tutorial / how to. You have this in here, at the start of you document. This should instead be located at the end of your document, after you have declared the policy. You seem to be missing instructions on how to actually running the unit tests?

Finally, your document has several small grammar mistakes and inconsistent capitalization. Please revise.

File: python/emach/tests/README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# Testing guidelines for eMach
+
+## Introduction
+This markdown document discusses the testing guidelines to be followed by the eMach contributors. The eMach testing framework is based on [unittest framework](https://docs.python.org/3/library/unittest.html). `unittest` is part of standard Python distribution, and therefore the contributors are not required to install additional Python packages.
```
Comment (Bharat-Ramadas) [inline python/emach/tests/README.md:4]: remove "the" from "the eMach contributrors"

File: python/emach/tests/README.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+This document assumes that the contributor is familiar with the unittest framework. However, if you are new to the unittest framework, please check out this [explainer video](https://www.youtube.com/watch?v=6tNS--WetLI&ab_channel=CoreySchafer).
```
Comment (Bharat-Ramadas) [inline python/emach/tests/README.md:6]: maybe introductory video instead of explainer video?

**Review by Bharat-Ramadas — COMMENTED** [review]
Nice work @aimthiazz !
I think the markdown has a few grammatical errors so it might be nice to review it once again before we merge it in.
In addition to this, it would be nice to have a introductory paragraph that gives the motivation for unit tests, what it actually means, and why should a developer consider unit testing their code. 

Apart from these 2 points, I think the PR is ready to be merged in.

**Review by Bharat-Ramadas — APPROVED** [review]
Looks good!

**Review by elsevers — APPROVED** [review]
Looks good to me. Thanks @aimthiazz

**Pull request merged**